sdcm/logcollector.py

```diff
@@ -1060,13 +1076,15 @@ def __init__(self, test_id=None, test_dir=None, params=None):
         self.loader_set = []
         self.kubernetes_set = []
         self.sct_set = []
+        self.pt_report_set = []
         self.cluster_log_collectors = {
             ScyllaLogCollector: self.db_cluster,
             MonitorLogCollector: self.monitor_set,
             SirenManagerLogCollector: self.siren_manager_set,
             LoaderLogCollector: self.loader_set,
             SCTLogCollector: self.sct_set,
             JepsenLogCollector: self.loader_set,
+            ParallelTimelinesReportCollector: self.pt_report_set
```

Comment:
Please, add comma at the end of the line.

Reply:
Added